CPP/73. Set Matrix Zeroes.cpp

```diff
@@ -0,0 +1,38 @@
+class Solution {
+public:
+    void setZeroes(vector<vector<int>>& m) {
+        bool zc = false;
+        bool zr = false;
+        for (int i = 0; i < m.size(); i++) { //check the first column
```

Comment:
This problem is a perfect opportunity to practice some code refactoring. Each of the following for loops only has a single purpose and can be extracted out in methods. With a few methods in place, the `setZeroes` method will itself read like the algorithm. The comments on the right `//` are perfect candidates for the method which will have that `for` loop. Feel free to tweak.